fdbserver/SkipList.cpp

```diff
@@ -1413,140 +1031,92 @@ void miniConflictSetTest() {
 void skipListTest() {
 	printf("Skip list test\n");
 
-	//sse4Test();
-
-	//A test case that breaks the old operator<
-	//KeyInfo a( LiteralStringRef("hello"), true, false, true, -1 );
-	//KeyInfo b( LiteralStringRef("hello\0"), false, false, false, 0 );
+	// A test case that breaks the old operator<
```

Comment:
I left this here since I may do some work on `operator<` and would like to use the test case. I can delete this as well and just refer to the PR as needed.